k8s/postgresql/chart/postgresql/templates/manifests.yaml

```diff
@@ -91,6 +91,30 @@ spec:
             secretKeyRef:
               name: {{ .Release.Name }}-secret
               key: password
+{{ if .Values.metrics.enabled }}
```

Comment:
Should be intended like a code around it - like https://github.com/GoogleCloudPlatform/click-to-deploy/pull/465/files#diff-b271cca83bfaf281c03a241d4c69b8cfR111